Resources/Textures/_Harmony/Clothing/OuterClothing/Hardsuits/qmsuit.rsi/meta.json

```
@@ -0,0 +1,31 @@
+
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Sprites by rot_witch (Discord)",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "icon"
+    },
+    {
+      "name": "equipped-OUTERCLOTHING",
+      "directions": 4
+    },
+    {
+      "name": "equipped-OUTERCLOTHING-vox",
+      "directions": 4
+    },
+    {
+      "name": "inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "inhand-right",
+      "directions": 4
+    }
+  ]
+}
```

```suggestion
{
    "version": 1,
    "license": "CC-BY-SA-3.0",
    "copyright": "Made by rot_witch (Discord)",
    "size": {
        "x": 32,
        "y": 32
    },
    "states": [
        {
            "name": "icon"
        },
        {
            "name": "equipped-OUTERCLOTHING",
            "directions": 4
        },
        {
            "name": "equipped-OUTERCLOTHING-vox",
            "directions": 4
        },
        {
            "name": "inhand-left",
            "directions": 4
        },
        {
            "name": "inhand-right",
            "directions": 4
        }
    ]
}
```

Indentation, and also you're missing EOF (You need an empty line at the end of your files)